signalfx-tracing/signalfx-python-tracing/django/blog/blog/settings.py

```diff
@@ -0,0 +1,143 @@
+# IMPORTANT:
+# The only requirement for auto-instrumentation is to add
+# 'signalfx_tracing' to installed apps.
+
+import logging
+import os
+from .secret_settings import SECRET_KEY
+
+# Create and configure logger
+if not os.path.exists('log'):
+    os.makedirs('log')
+logging.basicConfig(level='DEBUG',
+                    filename="log/server_log.log",
```

Comment:
Any reason to log in a file and not stdout for this demo?

Reply:
To make the stdout less busy. I assume a file will be more useful in actual production (which I am taking as the use case)

Reply:
Switched to stdout